OddsVsEven.php

```diff
@@ -1,105 +0,0 @@
-<?php
-
-/**
-*  Procedural function which counts the number of zeros and the number of ones.
-*  Zeros represents evens soldiers and ones represents odds soldiers.
-*/
-function battle($battle_field_configuration)
```

Comment:
Your variable names are good.
There are many coding standards but most frameworks accept PSR standard.
I guess you should read about it (for example $onlyCamelCaseVariables, empty lines, statements conventions, etc.)